src/destinations/README.md

```diff
@@ -65,7 +66,7 @@ PostgresDestination({
       targetType: "varchar",
     },
   ],
-})
+});
```

Comment:
does this need the ; as our JS standard does not typically allow those to be used without flagging an issue.

Reply:
I nuked it from orbit just to be sure